src/garage/torch/algos/ddpg.py

```diff
@@ -0,0 +1,300 @@
+"""This modules creates a DDPG model in PyTorch."""
+from collections import deque
+
+from dowel import logger, tabular
+import numpy as np
+import torch
+
+from garage.np.algos.off_policy_rl_algorithm import OffPolicyRLAlgorithm
+
+
+class DDPG(OffPolicyRLAlgorithm):
+    """A DDPG model implemented with PyTorch.
+
+    DDPG, also known as Deep Deterministic Policy Gradient, uses actor-critic
+    method to optimize the policy and reward prediction. It uses a supervised
```

Comment:
q-function instead of reward prediction?

Reply:
Q-function